pkg/api/types.go

```diff
@@ -940,10 +940,13 @@ type ContainerStatus struct {
 	Ready bool `json:"ready"`
 	// Note that this is calculated from dead containers.  But those containers are subject to
 	// garbage collection.  This value will get capped at 5 by GC.
-	RestartCount int    `json:"restartCount"`
-	Image        string `json:"image"`
-	ImageID      string `json:"imageID"`
-	ContainerID  string `json:"containerID,omitempty"`
+	RestartCount      int              `json:"restartCount"`
+	Image             string           `json:"image"`
+	ImageID           string           `json:"imageID"`
+	ContainerID       string           `json:"containerID,omitempty"`
+	ImageCreationTime unversioned.Time `json:"imageCreationTime,omitempty"`
```

Comment:
I generally like to group things, but since we started flat, let's keep it flat and move to a nested structure in a future API version.

Put all image-related fields adjacent to each other, however.